pkg/cmd/create/create.go

```diff
@@ -351,13 +358,38 @@ func (o *Options) helmfileSyncPreview(envVars map[string]string) error {
 		Args: append(args, "sync"),
 		Env:  envVars,
 	}
-	_, err = o.CommandRunner(c)
-	if err != nil {
-		return errors.Wrapf(err, "failed to run helmfile sync")
+	_, syncErr := o.CommandRunner(c)
+	if syncErr != nil {
+		syncErr = errors.Wrapf(syncErr, "failed to run helmfile sync")
+
+		syncErr = o.ProcessHelmfileSyncTimeoutOrReturnOriginalError(syncErr)
+
+		return syncErr
 	}
+
 	return nil
 }
 
+func (o *Options) ProcessHelmfileSyncTimeoutOrReturnOriginalError(syncError error) error {
+	if strings.Contains(syncError.Error(), timedOut) {
+		log.Logger().Infof("detected a failure on the preview environment %s so looking for an erroring pod", o.PreviewNamespace)
+		_, podsInNs, err := pods.GetPods(o.KubeClient, o.PreviewNamespace, "")
+		if err != nil {
+			log.Logger().Errorf("failed to find pods in namespace %s: %s", o.PreviewNamespace, err.Error())
```

Comment:
In this case the original helmfile output (syncError) isn't shown to the user. I think it would be better if it was.